drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -741,30 +742,39 @@ def _get_zone_by_uid(self, uid):
     def _get_item_stats(self):
         """
         Returns a tuple representing the number of correctly-placed items,
-        and the total number of items that must be placed on the board (non-decoy items).
+        and the total number of items on the board (including decoy items).
         """
-        required_items, __, correct_items = self._get_item_raw_stats()
+        items = self._get_item_raw_stats()
 
-        return len(correct_items), len(required_items)
+        correct_count = len(items.correct_placed) + len(items.correct_decoy)
+        total_count = len(items.required) + len(items.decoy)
+
+        return correct_count, total_count
 
     def _get_item_raw_stats(self):
         """
-        Returns a 3-tuple containing required, placed and correct items.
+        Returns a named tuple containing required, decoy, placed, correct
+        placed, and correct decoy items.
 
         Returns:
-            tuple: (required_items, placed_items, correct_items)
-                * required_items - IDs of items that must be placed on the board
-                * placed_items - IDs of items actually placed on the board
-                * correct_items - IDs of items that were placed correctly
+            namedtuple: (required, placed, correct_placed, decoy, correct_decoy)
+                * required - IDs of items that must be placed on the board
+                * placed - IDs of items actually placed on the board
+                * correct_placed - IDs of items that were placed correctly
+                * decoy - IDs of decoy items
+                * correct_decoy - IDs of decoy items that were unplaced
         """
         all_items = [str(item['id']) for item in self.data['items']]
         item_state = self._get_item_state()
 
-        required_items = set(item_id for item_id in all_items if self._get_item_zones(int(item_id)) != [])
-        placed_items = set(item_id for item_id in all_items if item_id in item_state)
-        correct_items = set(item_id for item_id in placed_items if item_state[item_id]['correct'])
+        required = set(item_id for item_id in all_items if self._get_item_zones(int(item_id)) != [])
+        placed = set(item_id for item_id in all_items if item_id in item_state)
+        correct_placed = set(item_id for item_id in placed if item_state[item_id]['correct'])
+        decoy = set(item_id for item_id in all_items if self._get_item_zones(int(item_id)) == [])
```

Comment:
@arbrandes nit: convert `all_items` to set, than `decoy = all_items - required_items`